src/requests/cache/create_cache.rs

```diff
@@ -0,0 +1,62 @@
+use momento_protos::control_client;
+use tonic::Request;
+
+use crate::requests::cache::MomentoRequest;
+use crate::{utils, CacheClient, MomentoResult};
+
+/// Request to create a cache.
+///
+/// # Arguments
+///
+/// * `cache_name` - The name of the cache to create.
+///
+/// # Example
+/// Assumes that a CacheClient named `cache_client` has been created and is available.
+/// ```no_run
+/// # fn main() -> anyhow::Result<()> {
+/// # use momento_test_util::create_doctest_client;
+/// # tokio_test::block_on(async {
+/// use momento::requests::cache::create_cache::CreateCache;
+/// use momento::requests::cache::create_cache::CreateCacheRequest;
+/// # let (cache_client, cache_name) = create_doctest_client();
+///
+/// let create_cache_request = CreateCacheRequest::new(cache_name.to_string());
```

Comment:
Having to call to_string or when giving strings to these methods feels strange. I tried changing them to &str, but then I need to define a lifetime for the request struct. What is the idiomatic thing to do here?

Reply:
I don't think `to_string` is uncommon but I think there might also be a way to specify a signature that can accept both, using some kind of `Into` thing. I'll create a ticket to look into it but we don't need to block this pr on it.